src/pages/Home/Description.js

```diff
@@ -142,7 +142,8 @@ function Description() {
             <Baseline>
                 <Trans i18nKey="common:homeBaseline">
                     {/* prettier-ignore */}
-                    Joignez un proche en visio, en un clic, <span>gratuitement.</span>
+                    Joignez un proche en visio, en un clic,{' '}
+                    <span>gratuitement.</span>
```

Comment:
`{' '}` breaks the translation
If you check the  homepage `gratuitement` has disappeared